src/ParallelAlgorithms/Events/ObserveTimeStep.hpp

```diff
@@ -101,13 +130,20 @@ class ObserveTimeStep : public Event<EventRegistrars> {
         "without a preceding '/'."};
   };
 
+  struct OutputTime {
```

Comment:
How about `PrintTime` or `EnableLogging` ("output" sounds to me a bit like writing to a file)? Please wait for @nilsdeppe to comment on this so you don't have to rename this multiple times :) When renaming, remember to rename the member variable as well.

Reply:
Yea, I like `PrintTime` or maybe `PrintTimeToTerminal`. If me end up printing more we could rename to `EnableLogging`, but I think just `PrintTimeToTerminal` would be good for now :)

Reply:
Either sounds good to me. Since I think `Print` sort of implies "to terminal/stdout" in pretty much every programming language I would go with the shorter `PrintTime`. @nilsdeppe sound ok?

Reply:
I prefer the longer since it is explicit about that the info is _not_ being printed to a file

Reply:
ok

Reply:
Just want to point out that stdout is more likely to be a file than a terminal in real runs.

Reply:
What I don't love about `PrintTime` is that to me this sounds like you're choosing whether to print the time into the reduction file, not print it separately as an update :shrug: